mungegithub/github/github.go

```diff
@@ -72,6 +72,15 @@ var (
 	combinedStatusLifetime = 5 * time.Second
 )
 
+func suggestOauthScopes(resp *github.Response, err error) error {
```

Comment:
The example is from the cache, hence a 200. I just wanted to showcase that we cannot really figure out from the headers if a 403 is due to lack of permissions, when in a 200 the union of the two headers is an empty set.